Description: use multiarch include dirs

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

Index: pykde4-4.13.97/CMakeLists.txt
===================================================================
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -8,6 +8,7 @@ find_package(KDE4 4.9.4 REQUIRED)
 # Match what's used in the main macros
 cmake_policy(SET CMP0002 OLD)
 find_package(PythonLibrary)
+find_package(PythonLibs)
 
 include(KDE4Defaults)
 include(MacroLibrary)
@@ -80,6 +81,7 @@ set_package_properties(PolkitQt PROPERTI
                        "Required to build Python Polkit bindings")
 
 include_directories(
+    ${PYTHON_INCLUDE_DIRS}
     ${PYTHON_INCLUDE_PATH}
     ${SIP_INCLUDE_DIR}
     ${QT_INCLUDE_DIR}
